packages/fast-components-react-base/src/listbox/listbox.tsx

```diff
@@ -0,0 +1,286 @@
+import * as ReactDOM from "react-dom";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { ListboxClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import {
+    ListboxHandledProps,
+    ListboxProps,
+    ListboxUnhandledProps,
+} from "./listbox.props";
+import * as React from "react";
+import { KeyCodes } from "@microsoft/fast-web-utilities";
+import { get, inRange, invert } from "lodash-es";
+import { canUseDOM } from "exenv-es6";
+
+export interface ListboxState {
+    /**
+     * The index of the focusable child
+     */
+    focusIndex: number;
+    focussedItemId: string;
```

Comment:
Our standard for things like this is `active`, the idea of focus is an implementation detail. I would recommend that this is `activeIndex`, `activeItemId`

Reply:
This is exactly what we do in context menu.  I don't have strong feelings either way myself tho.